### PR TITLE
Created topic app

### PR DESCRIPTION
**Pull request by Silent777**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=738&view=detail&selectedIssue=LVPYTP-75)
* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=738&view=detail&selectedIssue=LVPYTP-79)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/topic/urls.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from django.conf.urls import url
+from .views import TopicView
+
+urlpatterns = [
+    url(r'^$', TopicView.as_view())
+]
```
Comment (lhalam) [inline eventually/topic/urls.py]: add andline

File: eventually/.pylintrc [hunk 2]
```diff
@@ -330,7 +330,7 @@ ignore-comments=yes
 ignore-docstrings=yes
 
 # Ignore imports when computing similarities.
-ignore-imports=no
+ignore-imports=yes
```
Comment (lhalam) [inline eventually/.pylintrc]: ???

File: eventually/topic/urls.py (hunk 1 above)
Comment (Silent777) [reply eventually/topic/urls.py]: added

File: eventually/.pylintrc (hunk 2 above)
Comment (Silent777) [reply eventually/.pylintrc]: i don`t know how to resolve duplicate code problem

**Review by redmyt — APPROVED** [review]

**Pull request merged**